userdocs/theme/partials/header.html

```diff
@@ -75,5 +75,14 @@
       }
       toggleSwitch.addEventListener('click', switchTheme);
     </script>
+    <!-- Global site tag (gtag.js) - Google Analytics -->
+    <script async src="https://www.googletagmanager.com/gtag/js?id=G-GBCDQ2QFTZ"></script>
```

Comment:
Is it ok for the `GA_MEASUREMENT_ID` to be public? Do we want to pass it as an env var instead? I'm not sure 🤔

Reply:
Nope that's completely fine